src/settings/Settings.js

```diff
@@ -89,6 +89,12 @@ export const SETTINGS = {
     //     // not use this for new settings.
     //     invertedSettingName: "my-negative-setting",
     // },
+    "feature_bridge_errors": {
```

Comment:
This needs to be documented in riot-web too. See https://github.com/vector-im/riot-web/pull/10418 for an example.